src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -19,7 +19,7 @@
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Martin Hasoň <martin.hason@gmail.com>
  */
-class XmlUtils
+final class XmlUtils
```

Comment:
Right, It's mentioned in "Our backwards Compatibility Promise" part of contribution notes.